src/_pytest/outcomes.py

```diff
@@ -17,7 +17,12 @@ class OutcomeException(BaseException):
         contain info about test and collection outcomes.
     """
 
-    def __init__(self, msg: Optional[str] = None, pytrace: bool = True) -> None:
+    def __init__(
+        self,
+        msg: Optional[str] = None,
+        pytrace: bool = True,
+        short_msg: Optional[str] = None,
```

Comment:
Perhaps change `short_message` into a keyword only argument?

Reply:
I assume you mean `short_msg`, makes sense then.